ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -109,6 +109,11 @@ tests:
   optional: true
   steps:
     workflow: stackrox-stackrox-e2e-job
+- always_run: false
```

Comment:
`always_run: false` means these tests are opt-in. Are you setting up tests for the first time and want to try things out or do you expect everything is done-done-done after this change?

What's the idea for OCP coverage for Scanner V4?

A JIRA reference and the PR description will help answer these questions so I strongly recommend not to ignore the possibilities to provide context.

Please make sure to familiarize with the [Pull Request checklist](https://docs.engineering.redhat.com/display/StackRox/Making+changes+to+OpenShift+CI#MakingchangestoOpenShiftCI-PullRequestchecklist). Depending on how final you want this change to be, you'd need to add: `merge-gke-scanner-v4-tests` in this file, an entry in `stackrox-stackrox-nightlies.yaml` file, and make updates to `stackrox-stackrox-release-x.y.yaml`.

Reply:
First time setting up, not supposed to be the last PR for this.

Regarding the checklist, I did `make jobs` yesterday and added the generated presubmit job description. The Makefile target also touched dozens of other files, because there seems to be some kind of migration "vsphere -> vsphere02" in progress. `make update` did the same bogus (at least: unrelated to this PR) modifications here plus it failed with an error:

```
ModuleNotFoundError: No module named 'yaml'
make[1]: *** [Makefile:98: boskos-config] Error 1
make[1]: Leaving directory '/Users/mclasmeier/go/src/github.com/openshift/release'
make: *** [Makefile:58: update] Error 2
```